packages/ui/src/components/Checkbox/Checkbox.stories.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { Meta, Story } from '@storybook/react';
+
+import { HStack, VStack } from '../Stack';
+import { Checkbox, CheckboxProps } from '.';
```

Comment:
💭 It could be better to import it from `./Checkbox`. It lowers our risk of dependency cycles